src/v2/es-utils.js

```diff
@@ -70,10 +70,16 @@ class ESClass {
   }
 
   async getTemplate(name) {
-    const response = await this.client
+    const exists = await this.client
       .indices
-      .getTemplate({ name: name });
-    return response;
+      .existsTemplate({ name: name });
+    if (exists) {
+      const response = await this.client
+        .indices
+        .getTemplate({ name: name });
+      return response;
+    }
+    return { error: { message: 'Not Found' }};
```

Comment:
Should this be a 404?

Reply:
I changed it to `return false;` and catching it in the calling function (/server.js#181-185)